cross/xau/Makefile

```diff
@@ -0,0 +1,26 @@
+PKG_NAME = libXau
```

Comment:
Usually we try to keep the folder name and the PKG_NAME the same. Please follow this rule/guideline for constancy.